File: source/src/models/MtiScoreRecord.ts
export interface MtiScoreRecord {
  id: number;
  imo_number: string;
  year: number;
  month: number;
  mti_score: number | null;
  vessel_score: number | null;
  reporting_score: number | null;
  voyages_score: number | null;
  emissions_score: number | null;
  sanctions_score: number | null;
  created_at: Date;
  updated_at: Date;
}

File: source/src/main/java/com/example/mti/model/MtiScoreRecord.java
package com.example.mti.model;

import java.time.Instant;

public record MtiScoreRecord(
        String imoNumber,
        int year,
        int month,
        Double mtiScore,
        Double vesselScore,
        Double reportingScore,
        Double voyagesScore,
        Double emissionsScore,
        Double sanctionsScore,
        Instant createdAt,
        Instant updatedAt
) {
}
